src/dashboards/videos/details.jsonnet

```diff
@@ -119,3 +120,88 @@ dashboard.new(
   ),
   gridPos={ x: 0, y: 0, w: 12, h: 9 }
 )
+.addPanel(
+  gaugePanel.new(
+    title='Views count by distinct sessions',
+    // It would be nice to use markdown inline code blocks but they appear as regular code blocks.
+    // This is fixed in v8.0.0-beta1 of grafana.
+    // See https://github.com/grafana/grafana/releases/tag/v8.0.0-beta1
+    description=|||
+      A view is counted when the user has started the video.
+
+      Note: We count a **view** only once in a session because the **video played** event
+      can be emitted multiple times while the video is played - for example:
+        - when we pause the video and then resume it
+        - when we pause the video, seek to the start and then resume it
```

Comment:
In the previous implementation, we defined a view as a played event that occurs during the first 30s of a video.

Reply:
I was hesitating about this as this might over estimate the view count (when we have multiple `played` events during the first 30 seconds they get counted. But on the other side using sessions we might under estimate the view count (for example: when we watch the entire video then seek to the start and watch it again  - it gets counted only once.

Do we add both approaches / select one or combine them ?

Reply:
I think we can consider multiple views in a single session as... multiple views! As you said: those are real views even if the video is played multiple times by the same user in the same session. One way to reduce overestimating the views counter is to decrease the time span from [0-30s] to maybe [0-20s]. We'll have to try multiple thresholds. WDYT?

Reply:
Good Idea for the thresholds) Updated using them with the start time span instead of unique sessions.